drivers/soundwire/generic_bandwidth_allocation.c

```diff
@@ -338,18 +338,56 @@ static bool is_clock_scaling_supported(struct sdw_bus *bus)
 	return true;
 }
 
+/**
+ * check_all_peripherals_connected: Check if all peripherals can use the lane
+ *
+ * @m_rt: Manager runtime
+ * @lane: Lane number
+ */
+static bool check_all_peripherals_connected(struct sdw_master_runtime *m_rt, unsigned int lane)
```

Comment:
the function name is really neutral, no way to tell what it is checking, can you clarify it a bit?

Reply:
The function is checking if the manager lane connects to all peripherals. So that all peripherals can use the manager lane. i.e, if one peripheral doesn't connect to that lane, the lane can not be used. Update the function description.

Reply:
but the code is checking if at least one peripheral is connected to the lane (which is passed as a parameter)
So it is like is_plane_used() / is_plane_have_connected_peripheral() type of check, no?

Reply:
right, that is an interesting query to name for sure...
is_lane_connected_to_all_peripherals() ?

Reply:
updated